README.md

```diff
@@ -142,3 +142,30 @@ an input and returns a str. By default, `RichHelpFormatter` sets the function to
 ```python
 RichHelpFormatter.group_name_formatter = str.title
 ```
+
+## Contributing
+Start by [forking the repo](https://github.com/hamdanal/rich-argparse/fork) and cloning your fork.
+
+For those using [VS Code](https://code.visualstudio.com/) there's [preconfigured settings](.vscode/) in the repo.
+
+### Environment Setup
+You can use `tox` to setup a virtual environment for `rich_argparse` and install dependencies.
+
+```sh
+pip3 install tox
+cd /path/to/rich_argparse
+tox --devenv venv
+```
+
+A good way to experiment with changes to this package is to use the local copy of `rich_argparse` as the `argparse` formatter in another project and then running that project's executable with `--help` as you make changes.
+
+To install the local copy of `rich_argparse` as a package in another project called `project_that_uses_argparse`:
+
+#### Using Poetry
+If you are using [poetry](https://python-poetry.org/) to manage `project_that_uses_argparse`'s dependencies you can add this line to the `tool.poetry.dependencies` section of `project_that_uses_argparse/pyproject.toml`:
+
+```toml
+[tool.poetry.dependencies]
+rich_argparse = {path = "/path/to/rich-argparse", develop = true}
+```
```

Comment:
Let's change this a bit. Instead of *"A good way to experiment with changes..."*, you can say *"If you want to experiment with the effect of the changes on your project..."*.
Also please remove all the poetry stuff, it is not a good idea to advice people to install the module temporarily like this. This install will trigger the dependency resolution of poetry and will change their lock file which at best is not what they want (plus it is slow) and at worst it could break their workflow.
What I would recommend in this case is either to copy `rich_argparse.py` file to the root of your project and experiment there, or to create a throw away script with some argparse.ArgumentParser and use it in your clone of rich_argparse

Reply:
if you copy the `rich_argparse.py` you lose all the benefits of making your edits in a git repo, which is less than ideal IMHO. i would hope that anyone who us actually going to do this understands that it will change their `poetry.lock` file (and they should be doing it on a branch, so breaking their workflow shouldn't really be a concern).

Reply:
also if someone makes edits by copying, they won't get the auto-formatting settings you have put in place for VS Code users (assuming they are using VS Code)

Reply:
Then my second recommendation still holds. The poetry section has to go anyway since it is also not relevant. Why give instructions for poetry and not pip/pipx/pdm/etc. Telling users how to do editable installs in their own projects is not a goal of contributing guide.

Reply:
I would give instructions for pip etc except i don't know how to. IMHO a contributing guide should make it as easy as possible for a user to get the project up and running in the way they choose. They should basically be able to copy/paste stuff from the `README`/`CONTRIBUTING`/whatever and be up and running. i had to spend time researching how to install `rich-argparse` as a local package so i'm just trying to save the next person that time.

Reply:
either way i changed the wording significantly; take a look when you have a chance.